### ISSUE TITLE
Language localisation

### ISSUE THREAD

**Issue by victorsndvg** [USRSE/rse-glossary#6]
Nice initiative!

it could be great to be able to translate it to other languages. I could help with spanish :)

**Comment by vsoch** [comment]
This is a great idea! And hey long time no see, good to see you!!

There are two ways (I can think of) to go about this - 

 - use separate files for each language, at the risk of them going out of sync
 - put sections for languages below each term in the same file

The second approach is likely better so files don't get forgotten or go out of sync, although it risks having a really huge file if it gets too big. What are your thoughts? How is it done in other places? If we do choose to do separate files, there could be some check that (minimally) a term in an "other language" file is present in english. How would we keep the files / definitions in sync?

**Comment by victorsndvg** [comment]
Good to see you too :)

Honestly, I don't have any experience on this ...

I think the first approach is better for a growing project, but if it's not going to be so big the second approach is ok for me too.

About synchronization, I think that English can be the default language, If other language is not synchronized then the default language should be shown for a particular term of the glossary (maybe with a message telling "This term was not yet translated to $LANGUAGE, please help us by opening an issue here ...")

What do you think?

**Comment by vsoch** [comment]
I love it! And I agree, we should develop in anticipation of growing size.

If you want to add a file to _data, perhaps `terms_<language>.yml` and start with a copy paste of the file (and then writing the definitions) I can then work on the interface to make it easy to toggle between.

**Comment by victorsndvg** [comment]
Ok, perfect.

I will contribute soon with spanish (`es-ES`) and galician (`gl`).

We can use codes from http://www.lingoes.net/en/translator/langcode.htm to identify each language, e.g. `terms_es-ES.yml`. Do you agree?

**Comment by vsoch** [comment]
Yes, and let's add a link to that in the README under some section for "How to add a Language"
